src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -343,6 +343,11 @@
 helps['keyvault storage remove'] = """
 type: command
 short-summary: Remove a Key Vault managed Azure Storage Account and all associated SAS definitions. This operation requires the storage/delete permission.
+examples:
+  - name: Remove a Key Vault managed Azure Storage Account and all associated SAS definitions (autogenerated)
+    text: |
+        az keyvault storage remove --name My --vault-name MyVault
```

Comment:
Name `My` doesn't look good, suggested change it to `MyStorageAccount`.

```suggestion
        az keyvault storage remove --name MyStorageAccount --vault-name MyVault
```